Add libyt inline script for covering_grid(uniform)

koarakawaii opened the pull request:
Add `libyt` inline script for `covering_grid` function, which can be used to extract field with a specified AMR level. The extracted field is then converted to `.npz` file and saved.

koarakawaii committed:
add libyt inline script for covering_grid(uniform)

hyschive reviewed (changes requested):
@koarakawaii Thanks for the PR! Could you also add an example script for analyzing the resulting `.npz` files?

koarakawaii commented:
@hyschive Sure! Which folder should I put the analyze script?

hyschive commented:
@koarakawaii Put both scripts at `example/test_problem/ELBDM/IsolatedHalo/` for now. Also, add some brief descriptions in the `README` file therein.

koarakawaii committed:
add libyt inline analysis example of function covering_grid under example/test_problem/ELBDM/IsolatedHalo/libyt_script/ , add extra analyzing scripts, and record procedures in README

koarakawaii committed:
correct errors in submit script and remove example/libyt_inline_script/inline_script_covering_grid.py

koarakawaii committed:
remove example/libyt_inline_script/inline_script_covering_grid.py

koarakawaii commented:
@hyschive I have removed the `example/libyt_inline_script/inline_script_covering_grid.py` and added the necessary analysis scripts under `example/test_problem/ELBDM/IsolatedHalo/libyt_script/` .
